### DISCUSSION TITLE
mtex: a set in the ipf map

### DISCUSSION THREAD

**Discussion by lyg1232** [mtex-toolbox/mtex#1453] [Ask Anything]
Hello! I would like to ask, how to take the selected white border grains as shown in the figure below, and draw them as a set in the ipf map separately.
Thank you !
![438678914578465590](https://user-images.githubusercontent.com/81472625/173187060-d07e2b87-d0a8-43e2-ae85-820bc2ed03dd.jpg)

**Comment by kilir** [comment]
Hi
something like `grains('id',[65 765 81])` should work. However if you do not want to assign the id manually have a look at `help grain2d/findByLocation`.

Hope that helps.
Cheers,
Rüdiger
